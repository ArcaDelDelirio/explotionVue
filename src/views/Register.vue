<script>
export default {
  name: 'Register',
};
</script>
<template>
  <form action>
    <div class="container">
      <label>
        <b>email</b>
      </label>
      <input type="text" placeholder="Enter email" name="email" required />

      <label>
        <b>Username</b>
      </label>
      <input type="text" placeholder="Enter Username" name="uname" required />

      <label>
        <b>Password</b>
      </label>
      <input type="password" placeholder="Enter Password" name="psw" required />

      <button type="submit">Sign Up</button>
      <label>
        <input type="checkbox" checked="checked" name="remember" /> Remember me
      </label>
    </div>
  </form>
</template>


<style scoped>
</style>