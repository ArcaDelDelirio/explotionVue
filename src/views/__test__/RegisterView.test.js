import { mount } from '@vue/test-utils';
import Register from '../Register';

describe('Register View', () => {
  it('Render the component', () => {
    const wrapper = mount(Register);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
