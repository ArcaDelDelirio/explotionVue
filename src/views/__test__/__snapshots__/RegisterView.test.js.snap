// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register View Render the component 1`] = `
<form action="">
  <div class="container"><label><b>email</b></label> <input type="text" placeholder="Enter email" name="email" required="required"> <label><b>Username</b></label> <input type="text" placeholder="Enter Username" name="uname" required="required"> <label><b>Password</b></label> <input type="password" placeholder="Enter Password" name="psw" required="required"> <button type="submit">Sign Up</button> <label><input type="checkbox" checked="checked" name="remember"> Remember me
    </label></div>
</form>
`;
